qemu/tests/timedrift_check_after_load_vm.py

```diff
@@ -0,0 +1,163 @@
+                              % module_param, logging.info)
+        check_modules = arch.get_kvm_module_list()
+        error_context.context("check_module: '%s'" % check_modules, logging.info)
+        check_modules.reverse()
+        for module in check_modules:
+            rm_mod_cmd = "modprobe -r %s" % module
+            process.system(rm_mod_cmd, shell=True)
+        check_modules.reverse()
+        for module in check_modules:
+            load_mod_cmd = "modprobe %s" % module
+            if module == "kvm":
+                load_mod_cmd = "%s kvmclock_periodic_sync=%s" % (load_mod_cmd, module_param)
+            process.system(load_mod_cmd, shell=True)
+        check_mod_cmd = params["check_mod_cmd"]
+        if process.system_output(check_mod_cmd) != module_param:
+            test.fail("Cannot load kvm module with kvmclock_periodic_sync=%s"
+                      % module_param)
+
+    def setup():
+        """
+        On host, load kvm module with "kvmclock_periodic_sync=N"
+        sync time with ntp server and boot the guest
+        """
+        if arch.ARCH not in ('ppc64', 'ppc64le'):
+            _load_kvm_module_with_kvmclock_periodic_sync("N")
+        error_context.context("Sync host time with ntp server", logging.info)
+        ntp_cmd = params.get("ntp_cmd", "ntpdate clock.redhat.com")
+        status = process.system(ntp_cmd, shell=True)
+        if status:
+            test.cancel("Fail to sync host time with ntp server.")
+
+        error_context.context("Boot the guest", logging.info)
+        params["start_vm"] = "yes"
+        vm = env.get_vm(params["main_vm"])
+        env_process.preprocess_vm(test, params, env, vm.name)
+        vm.verify_alive()
+        return vm
+
+    def cleanup():
+        """
+        Close session and load kvm module with "kvmclock_periodic_sync=Y"
+        """
+        if session:
+            session.close()
+        env.unregister_vm(vm.name)
+        vm.destroy(gracefully=False, free_mac_addresses=True)
+        if arch.ARCH not in ('ppc64', 'ppc64le'):
+            _load_kvm_module_with_kvmclock_periodic_sync("Y")
+
+    def create_gagent():
+        """
+        Create a QemuAgent object to send guest agent command
+        """
+        gagent_serial_type = params["gagent_serial_type"]
+        gagent_name = params["gagent_name"]
+        if gagent_serial_type == guest_agent.QemuAgent.SERIAL_TYPE_VIRTIO:
+            filename = vm.get_virtio_port_filename(gagent_name)
+        elif gagent_serial_type == guest_agent.QemuAgent.SERIAL_TYPE_ISA:
+            filename = vm.get_serial_console_filename(gagent_name)
+        else:
+            raise guest_agent.VAgentNotSupportedError("Not supported serial"
+                                                      "type")
+        return guest_agent.QemuAgent(vm, gagent_name, gagent_serial_type,
+                                     filename, get_supported_cmds=True)
+
+    def run_qmp_cmd(qmp_port, qmp_cmd):
```

Comment:
This function is not necessary, send `qmp_port.send_args_cmd(qmp_cmd)` directly will be enough.